src/secrets/components/SecretsTab.tsx

```diff
@@ -0,0 +1,178 @@
+// Libraries
+import React, {FC, useState} from 'react'
+import {connect, ConnectedProps} from 'react-redux'
+
+// Components
+import {
+  Button,
+  ComponentColor,
+  ComponentSize,
+  EmptyState,
+  IconFont,
+  Sort,
+} from '@influxdata/clockface'
+import SearchWidget from 'src/shared/components/search_widget/SearchWidget'
+import TabbedPageHeader from 'src/shared/components/tabbed_page/TabbedPageHeader'
+import SecretsList from 'src/secrets/components/SecretsList'
+import FilterList from 'src/shared/components/FilterList'
+import ResourceSortDropdown from 'src/shared/components/resource_sort_dropdown/ResourceSortDropdown'
+import GetResources from 'src/resources/components/GetResources'
+
+// Selectors
+import {getAll} from 'src/resources/selectors'
+
+// Types
+import {AppState, ResourceType, Secret} from 'src/types'
+import {SortTypes} from 'src/shared/utils/sort'
+import {SecretSortKey} from 'src/shared/components/resource_sort_dropdown/generateSortItems'
+
+interface OwnProps {
+  secrets: Secret[]
+  searchTerm?: string
+  // importOverlayState?: OverlayState
+  sortDirection?: Sort
+  sortType?: SortTypes
+}
+
+type ReduxProps = ConnectedProps<typeof connector>
+type Props = OwnProps & ReduxProps
+
+const SecretsTab: FC<Props> = ({
+  secrets,
+  // importOverlayState = OverlayState.Closed,
```

Comment:
is there a plan to integrate overlays for this feature?

Reply:
The Overlay will be used for creating new secrets which is coming next.  I thought about getting rid of these commented bits for cleanliness's sake but they won't be around in this state for more than a few days so it seemed silly to take them all out to commit them and then put them all back in so I can use them again.  But if you have strong opinions about it I can remove the comments for now.

Reply:
nah, all good.